fs/config/config.go

```diff
@@ -61,7 +61,7 @@ const (
 // Global
 var (
 	// configData is the config file data structure
```

Comment:
Perhaps this for the comment? (it has the variable name wrong at the moment!)

```go
// configFile is the global config data structure. Don't use directly, use getConfigData()
```

Reply:
ah, yes, good catch!